New evaluation.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation</title>
+    
+    
+<link rel="stylesheet" href="css/New%20Evaluation.css">
+</head>
+<body>
+    
+    <form id="candidate">
+        <input type="text" placeholder="Candidate" name="Candidate">
+        <input type="text" placeholder="Interviewer" name="Interviewer">
+        <input type="date">
+    </form>
+    
+    <h1>Techinical Level</h1>
+<form>
+    <table align="centre">
+        <tr>
+            <th>Trainee</th>
+            <th>Junior</th>
+            <th>Mid</th>
+            <th>Senior</th>
+        </tr>
+        <tr>
+            <td>
+                <input type="radio" name="level" value="1"></td>
+            <td>
+                <input type="radio" name="level" value="j1">
+                <input type="radio" name="level" value="j2">
+                <input type="radio" name="level" value="j3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="m1">
+                <input type="radio" name="level" value="m2">
+                <input type="radio" name="level" value="m3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="s1">
+                <input type="radio" name="level" value="s2">
+                <input type="radio" name="level" value="s3">
+            </td>
+        </tr>
+        
+    
+    </table>
+</form>
+    
+    <h2>Should the candidate be hired?</h2>
+    <form>
+        <textarea rows="5" cols="30" placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance required for the candidate"></textarea>
+    </form>
+    
+    <h2>General Impression</h2>
+    <form>
+        <textarea rows="5" cols="30" placeholder="*required"></textarea>
+    </form>
+    
+    <h2>Workflow, Leadership &#38 Soft Skills</h2>
```

Comment:
Interesting entity, this `&#38`. What does it represent? And what could be up with it, such that it's displayed on a red background by `github`?